homeassistant/components/konnected/const.py

```diff
@@ -18,8 +18,24 @@
 STATE_LOW = "low"
 STATE_HIGH = "high"
 
-PIN_TO_ZONE = {1: 1, 2: 2, 5: 3, 6: 4, 7: 5, 8: "out", 9: 6}
-ZONE_TO_PIN = {zone: pin for pin, zone in PIN_TO_ZONE.items()}
+ZONES = [
```

Comment:
is there still a mapping needed for ESP8266 support?

Reply:
Yep - I still need to test the 8266.  Will have a few more changes coming. I just wanted to get this in front of you early since it's so large